src/DetailsView/components/export-dialog.tsx

```diff
@@ -4,6 +4,7 @@ import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
 import { Dialog, DialogFooter, DialogType } from 'office-ui-fabric-react/lib/Dialog';
 import { TextField } from 'office-ui-fabric-react/lib/TextField';
 import * as React from 'react';
+import { WindowUtils } from 'src/common/window-utils';
```

Comment:
This path needs to be relative. I am not sure why this is not failing the PR build.